lib/std/time.zig

```diff
@@ -50,51 +52,45 @@ pub fn sleep(nanoseconds: u64) void {
     std.os.nanosleep(s, ns);
 }
 
+test "sleep" {
+    sleep(1);
+}
+
 /// Get a calendar timestamp, in seconds, relative to UTC 1970-01-01.
 /// Precision of timing depends on the hardware and operating system.
-/// The return value is signed because it is possible to have a date that is
-/// before the epoch.
 /// See `std.os.clock_gettime` for a POSIX timestamp.
-pub fn timestamp() i64 {
+pub fn timestamp() u64 {
```

Comment:
Originally, the code looked like it could only go signed for windows which has a farther back epoch. Doing a bit more research though, it can go before the epoch (?) and is normally kept signed even for POSIX platforms. So i'll change it back.

* https://github.com/abseil/abseil-cpp/blob/1ae9b71c474628d60eb251a3f62967fe64151bb2/absl/time/internal/get_current_time_posix.inc#L13
* https://github.com/golang/go/blob/master/src/time/time.go#L128-L140
* https://doc.rust-lang.org/src/std/sys/unix/time.rs.html#12-14